content/features/introductionToFeatures/chap3/import_character.md

```diff
@@ -20,7 +20,7 @@ The *Dude* model is one that has been built with its own skeleton animation.
 Once imported the character and its skeleton are obtained from the meshes and skeletons properties of the results object.
 
 ```javascript
-BABYLON.SceneLoader.ImportMeshAsync("mesh name", "path to model", "model file", scene).then((result) => {
+BABYLON.SceneLoader.ImportMeshAsync("him" /* mesh name */, "/scenes/Dude/" /* path to model */, "Dude.babylon" /* model file */, scene).then((result) => {

```

Comment:
So far all JS code can be copy pasted and it works, but this one was using the strings to explain the parameters, I added the comments so the code can be copy pasted, but still there is an explanation. I can discard if not appropriate.